### DISCUSSION TITLE
Turn on and off Tradfri Ikea lights with a physical not smart switch does not update the state

### DISCUSSION THREAD

**Discussion by salvovitale** [Koenkk/zigbee2mqtt#9379] [Devices]
Hi all, 
recently i migrated all my zigbee devices from deConz to zigbee2mqtt. I must say that I am quite happy. Smother interface, better paring procedure, better devices interface, more devices supported, etc... . However, if there was something that deConz was doing better that is the synchronisation with power on/off for Ikea Tradfri lights (it may be also with other lights but I only tested it with the Tradfri lights).

I currently am using these ikea lights 
- TRADFRI bulb E27 WW 806lm
- TRADFRI bulb E27 WW clear 250lm
- TRADFRI bulb GU10 WW 400lm

The issue is that when i am powering off/on the light using my physical non smart switch I do not see any state update in zigbee2mqtt. This of course creates all sort of confusion with my HomeAssistant interface/automations. Basically, a lights that is on and with State:"On", when powered off still shows State:"On". 

Is there a solution to this problem? Am I setting things wrongly?

**Comment by Koenkk** [comment]
I would suggest to use the re-implemented availability feature which will launch with the next release on 1 November.

**Reply by salvovitale** [reply]
@Koenkk  thank you for your fast reply. I ll check that.
